task26.py

```diff
@@ -0,0 +1,13 @@
+'''Задача 26:  Напишите программу, которая на вход принимает два числа A и B, и возводит число А в целую степень B с помощью рекурсии.'''
+
+a = int(input('Введите число А: '))
+b = int(input('Введите степень числа A: '))
+
+def stp(a, b):
+    if a == 0:
+        return ('Выражение не имеет смысла')
+    elif b == 0:
+        return 1
+    else:
+        return a * stp(a, b - 1)

```

Comment:
`else` можно не использовать, так как `return` завершает работу функции
```suggestion
    if b == 0:
        return 1

    return a * stp(a, b - 1)

```